test/reporters/tests/custom-reporter.spec.ts

```diff
@@ -19,5 +18,34 @@ test('custom reporters work', async() => {
     windowsHide: false,
   })
 
-  expect(stdout).toContain('hello from custom reporter')
-}, 40000)
+  return stdout
+}
+
+describe('Custom reporters', () => {
```

Comment:
It would also be great to see a test case for "custom reporter defined in configuration using path".

```ts
  test: {
    reporters: ['./src/custom-reporter'],
  }
```

Reply:
Sure I don't mind adding that

Reply:
@AriPerkkio the first test case is one that uses a custom reporter defined in a config file

Reply:
Sure, that one is a reporter instance instead of a string path.

https://github.com/vitest-dev/vitest/blob/0c9137d9d3267be860a59c619e2eee981e51a64b/test/reporters/custom-reporter.vitest.config.ts#L19

I don't think anything covers "custom reporter defined in configuration **using path**" yet.